### ISSUE TITLE
Single-sentence and cross-sentence

### ISSUE THREAD

**Issue by LiyingCheng95** [VioletPeng/GraphLSTM_release#6]
Noticed that the test sets are different for single-sentence and cross-sentence. 
(1) How did you determine the single-sentence instances? Did you select the instances with only one "." as single-sentence? Can you provide that partial test datasets? 
(2) Did you use the same training data for these two settings?
(3) How did you split the dataset into train/dev/test? Do you mind sharing the train/dev/test set?

Thanks a lot!

**Comment by VioletPeng** [comment]
They should be the same set, and we just break the instances down into single-sentence instances (all entity mentions appeared in the same sentence) and cross-sentence instances (the entity mentions appeared in different sentences) within the same test set. They are mutually exclusive by construction but they are from the same test set.
Why do you think they are different?
We use a cross-validation setting, so we have 5 splits, we always train on 4 and test on the rest 1 split.
